outlines/models/openai.py

```diff
@@ -327,7 +334,6 @@ async def generate_chat(
 
     """
 
-    @cache()
```

Comment:
Why making this change? This avoid having to define an intermediate function `cached_call`